src/components/Icon/Icon.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import '../../../node_modules/@fortawesome/fontawesome-free/css/all.css'
+// import { fontawesomeIconList } from './iconList'
+
+export const Icon = ({
+  name,
+  size = `1x`,
+  animation = null,
+  rotate = null,
+  border = false
+}) => {
+  const options = (name, size, animation, rotate, border) => {
+    const optionArray = [`fa-${name}`, `fa-${size}`]
+    if (animation) {
+      optionArray.push(`fa-${animation}`)
+    }
+    if (rotate) {
+      optionArray.push(`fa-${rotate}`)
+    }
+    if (border) {
+      optionArray.push(`fa-border`)
+    }
+    return optionArray.join(` `)
+  }
```

Comment:
- I think if we name it something like `getOptions` that could be nice—although this is purely my personal preference, I usually like to name my functions starting with verbs.

- Instead of this function taking 5 ordered arguments, it could be nice if we used an _object_ as a single argument. That way we don't have to worry about the order of the arguments when calling the function, and it's easy to trace. 
  ```js
  const options = ({ name, size, animation, rotate, border }) => {
    // rest of the function
  }

  // and then using it
  options({ name, size, animation, rotate, border })
  ```